x-pack/plugins/telemetry_collection_xpack/schema/xpack_plugins.json

```diff
@@ -2219,13 +2219,6 @@
         }
       }
     },
-    "fileUploadTelemetry": {
-      "properties": {
-        "filesUploadedTotalCount": {
```

Comment:
This metric is no longer collected? I suspect we want to add it to the `fileUpload` collector?

Reply:
Its collected but is called `file_upload.index_creation_count`.

There were 2 copies of `import` endpoint. One in ml and one used by maps. Each copy had its own telemetry. The goal of this project was to combine into a single endpoint so now there is just a single telemetry.

Next steps are to get better telemetry like which application file was uploaded from and which file extension was uploaded, but that is out of scope for this PR.

Reply:
Thanks for clarifying!